src/Microsoft.OData.Core/ODataBatchReader.cs

```diff
@@ -17,24 +17,23 @@ namespace Microsoft.OData
     #endregion Namespaces
 
     /// <summary>
-    /// Class for reading OData batch messages; also verifies the proper sequence of read calls on the reader.
+    /// Abstract Class for reading OData batch messages; also verifies the proper sequence of read calls on the reader.
     /// </summary>
-    public sealed class ODataBatchReader : IODataBatchOperationListener
+    public abstract class ODataBatchReader : IODataBatchOperationListener

```

Comment:
>abstract [](start = 11, length = 8)

Is this breaking change.  I see tests that were initializing the reader and writer. Need to analyzed more of usage patterns